.github/workflows/run-litmus.yml

```diff
@@ -0,0 +1,45 @@
+name: Run litmus tests on different platforms
+
+on:
+  push:
+    branches:
+      - dev-gh-ci
+      - development
+      - main
+
+jobs:
+  linux-run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v4
+        with:
+          distribution: oracle
+          java-version: 17
+      - run: chmod +x gradlew
+      - name: Assemble CLI binary
+        run: ./gradlew cli:linkReleaseExecutableLinuxX64
+      - name: Run litmus tests via CLI
+        run: ./cli/build/bin/linuxX64/releaseExecutable/cli.kexe -r pthread ".*"
+      - name: Run a single test with JCStress
+        run: ./gradlew :cli:jvmRun --args="-r jcstress -j '-m quick' StoreBuffering.Plain"
```

Comment:
Why not run all jcstress tests?

Reply:
@DLochmelis33 ping

Reply:
Discussed this on a meeting. 

The problem is that apparently there is no option in jcstress to control the duration of testing, except the `-m` (mode) option. But, event with `-m quick` all the tests that we have now are run ~30mins, which is too long. 

The purpose of our local CI is to run "smoke-tests" of the `litmuskt` tool itself (i.e. to quickly check the basic functionality of the tool), and not to exhaustively test the Kotlin compiler.
Therefore, it was decided to run just a single jcstress test on CI, only to check that the jcstress integration works.

Reply:
@DLochmelis33 could you please add the info from the comment above to the code. 
After this we can merge this PR.